src/transform.ts

```diff
@@ -1,7 +1,7 @@
 import { camelCase } from 'change-case'
-import { v2 } from '@date-fns/upgrade'
-import transformImports from "transform-imports";
-import * as functionData from "./data/functionData.json";
+import { convertTokens } from '@date-fns/upgrade/v2'
+import transformImports from 'transform-imports'
```

Comment:
Guess we need to clean up eslint/prettier config in this repo.

Reply:
That would be cool, I used Prettier with date-fns settings.